Fix: Update config-store.md for decK tab steps (#8648)
Kong/docs.konghq.com: app/konnect/gateway-manager/configuration/config-store.md

{% endnavtab %}
{% navtab decK %}

> Create a config store entity in {{site.konnect_short_name}} and save the `config_store_id` from the response body.

Reviewer: ```suggestion
Create a config store entity in {{site.konnect_short_name}} and save the `config_store_id` from the response body:
```

> Create a config store entity in {{site.konnect_short_name}} and save the `config_store_id` from the response body:

Reviewer: ```suggestion
decK doesn't support creating a {{site.konnect_short_name}} config store, but you can reference secrets stored in a config store with decK.

Using the Control Plane Config API, create a config store entity in {{site.konnect_short_name}} and save the `config_store_id` from the response body:
```

```sh 
curl -i -X POST https://{region}.api.konghq.com/v2/control-planes/{control-plane-id}/config-stores \
  --header 'Authorization: Bearer {kpat_token}' \
  --header 'Content-Type: application/json' \
  --data '{
	"name": "my-config-store"
}'
```

> Using the `config_store_id` create a `POST` request to associate the config store with the vault.

Reviewer: ```suggestion
Using the generated `config_store_id`, send a `POST` request to associate the config store with the vault:
```

    
```sh
curl -i -X POST https://{region}.api.konghq.com/v2/control-planes/{control-plane-id}/core-entities/vaults/  \
  --header 'Authorization: Bearer {kpat_token}' \
  --header 'Content-Type: application/json' \
  --data '{
	"config":{
		"config_store_id": "{my-config-store-id}"
	},
	"description": "Description of your vault",
	"name": "konnect",
	"prefix": "mysecretvault"
}'
```

> Link the decK YAML file with the {{site.konnect_short_name}} Config Store.

Reviewer: ```suggestion
Reference the {{site.konnect_short_name}} config store in your decK file:
```
